themes/openy_themes/openy_carnation/openy_carnation.theme

```diff
@@ -461,13 +455,6 @@ function openy_carnation_form_system_theme_settings_alter(&$form, FormStateInter
     '#open' => TRUE,
   ];
 
-  $form['search']['display_search_form'] = [
```

Comment:
@dchubar-fj I don't think we should remove the configuration form. I believe it would be better if the modules enable this setting.

Reply:
@AndreyMaximov Other themes don't have such settings,  I think all theme should work the same way. 
Do you think we should have an additional layer of settings here? So if you want to add search to the website you have to enable the search module and then enable displaying the search form here? Why?
I guess Open Y users are not familiar with Drupal a lot and everything should be easier.

Reply:
Reverted configuraition form